docs/tutorials/00-getting-started.md

```diff
@@ -1,17 +1,40 @@
-# Getting Started
-
 ## Installation
 
 ```bash
 npm install bigscreen-player --save
 ```
 
-## Configuration
+## Get To Playback
+
+For a start, let's play back an on-demand video.
+
+To start a playback session you must minimally provide:
+
+- Some configuration. See [Configuration](#configuration)
+- A streaming manifest. See [Minimal Data](#minimal-data)
+- A HTML element to render playback. See [Initialisation](#initialisation)
+
+### Configuration
 
-Bigscreen Player has some global configuration that is needed before initialisation. A *playback strategy* must be configured:
+You must provide a *playback strategy* to use BigscreenPlayer:
 
 ```javascript
-window.bigscreenPlayer.playbackStrategy = 'msestrategy' // OR 'nativestrategy' OR 'hybridstrategy' OR 'basicstrategy'
+window.bigscreenPlayer.playbackStrategy = 'msestrategy' // OR 'nativestrategy' OR 'basicstrategy'
+```
+
+The MSEStrategy uses DASH. It is most likely what you want. More detail in the [documentation on playback strategies](<https://bbc.github.io/bigscreen-player/api/tutorial-playback-strategies.html>).
```

Comment:
It's weird I know but the URL is https://bbc.github.io/bigscreen-player/api/tutorial-Playback%20Strategies.html, with a space in it 

```diff
- The MSEStrategy uses DASH. It is most likely what you want. More detail in the [documentation on playback strategies](<https://bbc.github.io/bigscreen-player/api/tutorial-playback-strategies.html>).
+ The MSEStrategy uses DASH. It is most likely what you want. More detail in the [documentation on playback strategies](<https://bbc.github.io/bigscreen-player/api/tutorial-Playback%20Strategies.html>).
```

Reply:
Not since I renamed that file 😛

Reply:
My bad :sweat_smile: